app/com/linkedin/drelephant/spark/SparkMetricsAggregator.scala

```diff
@@ -63,10 +63,18 @@ class SparkMetricsAggregator(private val aggregatorConfigurationData: Aggregator
       case false => 0.0
```

Comment:
This change is not related to fetchers, maybe there should be a separate PR for this change with more context.